jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecCommandExecutable.kt

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.ecs.exec
+
+import com.intellij.openapi.util.SystemInfo
+import com.intellij.util.text.SemVer
+import software.aws.toolkits.jetbrains.core.executables.AutoResolvable
+import software.aws.toolkits.jetbrains.core.executables.ExecutableCommon
+import software.aws.toolkits.jetbrains.core.executables.ExecutableType
+import software.aws.toolkits.jetbrains.core.executables.Validatable
+import software.aws.toolkits.jetbrains.settings.ExecutableDetector
+import java.nio.file.Path
+import java.nio.file.Paths
+
+class EcsExecCommandExecutable : ExecutableType<SemVer>, AutoResolvable, Validatable {
```

Comment:
should probably be `AWS CLI` and not `ecs exec`, since it looks like we're just expecting `session-manager-plugin` to be on the path somewhere (and that part is not configurable)

Reply:
Changed it AWS CLI